telegram/_message.py

```diff
@@ -367,7 +405,7 @@ class Message(TelegramObject):
 
             .. versionadded:: 20.0
         reply_markup (:class:`telegram.InlineKeyboardMarkup`): Optional. Inline keyboard attached
-            to the message.
+            to the message. ``login_url`` buttons are represented as ordinary url buttons.
```

Comment:
```suggestion
            to the message. :paramref:`~telegram.InlineKeyboardButton.login_url` buttons are represented as ordinary url buttons.
```
same for Args section.

Reply:
done